ldsc/src/main/resources/install-ldsc.sh

```diff
@@ -25,7 +25,7 @@ sudo pip2 install -U pandas
 sudo pip2 install -U numpy
 
 # download the 1000g LD data for ancestries
-aws s3 cp s3://dig-analysis-data/bin/clumping/ . --recursive
+#aws s3 cp s3://dig-analysis-data/bin/clumping/ . --recursive
```

Comment:
If this isn't necessary I would just delete it. I can't quite find a place where it is used in any of the code.

Reply:
If no other bits use, okay to delete!